applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -22,11 +23,23 @@
          ,delete/2
         ]).
 
--include("crossbar.hrl").
+-include_lib("crossbar/src/crossbar.hrl").
```

Comment:
@lazedo: i want to change them all eventually. with -include sync and other similar tools are unable to recompile the module (they don't know where to searh for hrl, but with include_lib they know (code:lib_dir(app)). this works nicely with relx development releases.